src/RA_AchievementSet.cpp

```diff
@@ -311,7 +312,7 @@ BOOL AchievementSet::SaveToFile()
     //}
 }
 
-BOOL AchievementSet::Serialize(FileStream& Stream)
+BOOL AchievementSet::Serialize(_UNUSED std::iostream& Stream)
```

Comment:
New rapidjson has FileRead/WriteStream, the iostream wrappers should be fast if you don't use `std::endl`. 
`std::ios::sync_with_stdio(false)` might help if using stuff like `cout`.

I just put iostream and marked the function as deprecated since it's not used anymore.